OpenTabletDriver.Plugin/Tablet/Wheel/IRelativeWheelReport.cs

```diff
@@ -0,0 +1,13 @@
+namespace OpenTabletDriver.Plugin.Tablet.Wheel
+{
+    /// <summary>
+    /// An auxiliary report containing states of a wheel/ring input.
+    /// </summary>
+    public interface IRelativeWheelReport : IDeviceReport
+    {
+        /// <summary>
+        /// The relative wheel movement from its previous state, or 0 to indicate an absence of touch.
```

Comment:
```suggestion
        /// The relative wheel movement from its previous state, 0 to indicate an absence of movement or null for an absence of touch.
```

Got skill issued by github not allowing multiple suggestions per lines & not being able to figure out how to add on top of the suggestion (especially using this slow interface that keep resetting pending changes)